src/components/UserInformation.js

```diff
@@ -94,6 +110,16 @@ class UserInformation extends React.Component {
     }
   }
 
+  validate = () => {
+    if (this.state.usernameOption) {
```

Comment:
if i click the "add user" button without selecting an option in the dropdown, the form is submitted without validation